prelude/toolchains/msvc/vswhere.py

```diff
@@ -202,6 +203,74 @@ def write_tool_json(out, tool):
     out.write(j)
 
 
+# for use with the ewdk to grab the environment strings
+def get_ewdk_env(ewdkdir: Path):
+    """
+    Inspiration taken from the following:
+    http://pythonwise.blogspot.fr/2010/04/sourcing-shell-script.html (Miki Tebeka)
+    http://stackoverflow.com/questions/3503719/#comment28061110_3505826 (ahal)
+    """
+
+    output = None
+
+    # We need to write the script that will make the important variables available
+    with tempfile.NamedTemporaryFile(prefix="VcVarsExtract", suffix=".bat", mode="w", delete=False) as tmp:
+        print(f"temp file at {tmp.name}")
+        output = tmp.name
```

Comment:
It's either unused or need to be replaced in `cmd`. In latter case rename it to `wrapper_script` to avoid clash with `output`  from `check_output`.
```suggestion
```

Reply:
It was for debugging when it was getting cleaned up, and I missed it. Deleting now.